sklearn/ensemble/_hist_gradient_boosting/tests/test_binning.py

```diff
@@ -26,32 +26,33 @@
 
 def test_find_binning_thresholds_regular_data():
     data = np.linspace(0, 10, 1001)
-    bin_thresholds = _find_binning_thresholds(data, max_bins=10)
+    bin_thresholds = _find_binning_thresholds(data, sample_weight=None, max_bins=10)
     assert_allclose(bin_thresholds, [1, 2, 3, 4, 5, 6, 7, 8, 9])
 
-    bin_thresholds = _find_binning_thresholds(data, max_bins=5)
+    bin_thresholds = _find_binning_thresholds(data, sample_weight=None, max_bins=5)
     assert_allclose(bin_thresholds, [2, 4, 6, 8])
 
 
 def test_find_binning_thresholds_small_regular_data():
     data = np.linspace(0, 10, 11)
 
-    bin_thresholds = _find_binning_thresholds(data, max_bins=5)
+    bin_thresholds = _find_binning_thresholds(data, sample_weight=None, max_bins=5)
```

Comment:
Since this is the default, there is no point in changing those line. The code in `main` is fine, and the existing test should pass unchanged.

```suggestion
    bin_thresholds = _find_binning_thresholds(data, max_bins=5)
```